webapp/channels/src/components/admin_console/admin_definition.tsx

```diff
@@ -5833,6 +5833,25 @@ const AdminDefinition: AdminDefinitionType = {
                     ],
                 },
             },
+            embedding: {
+                url: 'integrations/embedding',
+                title: defineMessage({id: 'admin.sidebar.embedding', defaultMessage: 'Embedding'}),
+                isHidden: it.not(it.userHasReadPermissionOnResource(RESOURCE_KEYS.INTEGRATIONS.CORS)), // MM-63356: it.not(it.userHasReadPermissionOnResource(RESOURCE_KEYS.INTEGRATIONS.EMBEDDING)),
```

Comment:
Wrong permission? Actually, as a larger question, why do we need a new permission at all?

Reply:
All the settings seem to have discreet permissions.

Reply:
CORS is the closest conceptually but is different than CSP that frame ancestors falls under.  One is for allowing cross-domain resource access, while the latter allows embedding.  I'm fine with just using the CORS permission, but that goes against how the system console permissions works, where each page can be shown/hidden with a discrete permission.

Frankly there is a crazy amount of plumbing spread all over the app just to add a setting to the system console.

Reply:
I reverted the new permission - we'll just use CORS permission for now.